### ISSUE TITLE
is .eslintrc ignored?

### ISSUE THREAD

**Issue by gsabran** [roadhump/SublimeLinter-eslint#141]
I've followed the setup instructions. I've a .eslintrc at the root of my sublime project that is

```json
{
  "env": {
    "es6": true,
    "browser": true,
    "node": true
  },

  "plugins": [
    "react"
  ],

  "ecmaFeatures": {
    "modules": true,
    ...
  },

  "rules": {}
}
```

and it doesn't seem to be taken into account because 
`import {something} from 'somepackage';`
raises a lint error `import and export may appear only with sourcetype module` that should not be raised due to the ecmaFeatures settings.

**Comment by benmosher** [comment]
Does it work from the command line?

Also, in ESLint 2+ you specify modules as follows:

```json
{
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": 6 
  }
}
```

**Comment by gsabran** [comment]
Ah thanks. I think that renaming `.eslintrc` to `.eslintrc.js` made things work. Feel free to close the issue.
